### ISSUE TITLE
Remove app-id in internal structures of salix

### ISSUE THREAD

**Issue by tvdstorm** [cwi-swat/salix#12]

**Comment by joukestoel** [comment]
If I remember correctly this app-id is needed in the case of a page running multiple Salix apps but I might be mistaken

**Comment by tvdstorm** [comment]
Yes, but it should not be needed, as per the compositionality offered by salix itself.

**Comment by tvdstorm** [comment]
Are you depending on this?

**Comment by tvdstorm** [comment]
Update: multiple apps in one page was used in Bacata, but is now solved differently by @jurgenvinju
